moveit_ros/moveit_servo/include/moveit_servo/pose_tracking.h

```diff
@@ -87,7 +87,7 @@ class PoseTracking
 {
 public:
   /** \brief Constructor. Loads ROS parameters under the given namespace. */
-  PoseTracking(const rclcpp::Node::SharedPtr& node, const ServoParameters::SharedConstPtr& servo_parameters,
+  PoseTracking(const rclcpp::Node::SharedPtr& node, std::shared_ptr<const servo::ParamListener>& servo_param_listener,
```

Comment:
Any reason to have `shared_ptr` or can it be a `unique_ptr`?

Reply:
We are passing the `ParamListener` when creating the `Servo` object.
`unique_ptr` reference might be fine as long as the original copy stays in scope.
Which is the case for pose tracking demo.
Not sure if there will be a case were the `ParamListener` is created in a function and passed and the function goes out of scope and destroys the `ParamListener` `unique_ptr`